libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccountManager.java

```diff
@@ -350,23 +349,17 @@ public void switchToUser(UserAccount user, int userSwitchType, Bundle extras) {
 	 * in ClientManager will return a RestClient instance for the new user.
 	 */
 	public void switchToNewUser() {
-		final Bundle options = SalesforceSDKManager.getInstance().getLoginOptions().asBundle();
-		switchToNewUserWithOptions(options);
+		final Bundle options = new Bundle();
```

Comment:
This is a pattern you will see over and over but it strikes me a very odd.  The only value in`LoginOptions` that `LoginActivity` would read from a bundle is the `jwt` string 🤷.   That `jwt` value is not set on `LoginViewModel`.